ts/components/ui/Markdown/index.tsx

```diff
@@ -142,6 +183,7 @@ class Markdown extends React.PureComponent<Props, State> {
               style={webViewStyle}
               originWhitelist={["*"]}
               source={{ html, baseUrl: "" }}
+              allowFileAccess={true}
```

Comment:
uhm say somebody is able to inject javascript code in a message, will he be able to read the redux store (and the content of all messages?)

Reply:
it should be possible to use a data uri to embed the font in the css (*) and avoid giving access to the filesystem to the webview - see https://www.webucator.com/blog/2016/11/inline-web-font-avoid-fout/ and https://gist.github.com/viljamis/c4016ff88745a0846b94 and http://stephenscaff.com/articles/2013/09/font-face-and-base64-data-uri/

(*) this can be done at build time

Reply:
Before answering let me try to remove it and check if font loading still works.

Reply:
embedding the font in the css has the additional benefit of not needing RNFS

Reply:
I have removed the prop and it is still working on both iOS and Android.